ToNote/Models/Note.cs

```diff
@@ -27,5 +30,46 @@ public string Name
                 }
             }
         }
+
+        private string _Description;
+
+        public string Description
+        {
+            get => _Description;
+            set
+            {
+                if (_Description != value)
+                {
+                    _Description = value;
+
+                    RaisePropertyChanged(nameof(Description));
+                }
+            }
+        }
+
+        private List<string> _FileNames;
+
+        public List<string> FileNames
+        {
+            get => _FileNames;
+            set
+            {
+                if (_FileNames != value)
+                {
+                    _FileNames = value;
+
+                    RaisePropertyChanged(nameof(FileNames));
+                }
+            }
+        }
+
+        public void DeleteFile(string file)
+        {
+            if (File.Exists(file))
```

Comment:
Note is not responsible for creating its files, but is responsible for deleting them. I don't think this is good desing. Also, with current impl, you could delete a file unrelated to the actual note.
Maybe move the `if (File.Exists(file))
                File.Delete(file);` to the callsite and refactor this method to `RemoveNoteFile(string file)` or similar? Or even just move all of the logic to the callsite.

Reply:
It was decided that this is just a temporary solution, and thus will be refactored with a separate [enh](https://trello.com/c/Fxcjq3yp).